eng/targets/Settings.props

```diff
@@ -132,6 +132,13 @@
     <PackageReference Include="Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers" Version="$(MicrosoftCodeAnalysisPerformanceSensitiveAnalyzersVersion)" PrivateAssets="all" />
   </ItemGroup>
 
+  <!-- 
+     Code indexing targets to help generating LSIF from indexing builds.
+  -->
+  <ItemGroup>
+    <PackageReference Include="RichCodeNav.EnvVarDump" Version="0.1.1407-alpha" />
```

Comment:
This should probably be defining a version variable for the version number like we are doing for the packages above. The variable goes in eng/Versions.props.

Reply:
Fixed this in the 2nd commit.